scripts/build/termux_create_debian_subpackages.sh

```diff
@@ -60,6 +60,13 @@ termux_create_debian_subpackages() {
 		[ "$TERMUX_SUBPKG_PLATFORM_INDEPENDENT" = "true" ] && SUB_PKG_ARCH=all
 
 		cd "$SUB_PKG_DIR/massage"
+		# Check that files were actually installed, else don't subpackage.
+		if [ "$(find . -type f -print | head -n1)" = "" ]; then
+			echo "No files in subpackage '$SUB_PKG_NAME' when built for $SUB_PKG_ARCH with package '$TERMUX_PKG_NAME', so"
+			echo "the subpackage was not created. If unexpected, check to make sure the files are where you expect."
+			cd "$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX"
+			continue
+		fi
```

Comment:
How about only applying to platform-independent subpackages? That should be sufficient if I understand the description correctly.

Reply:
Also it should be possible to restrict to the case when the parent package is platform-*dependent*.

Reply:
Those conditions you suggest would be enough to allow these Swift subpackages, but do we generally want to allow empty subpackages? I took this check from [the massage step for regular packages, which enforces the even more stringent requirement that one non-doc file _must_ be installed](https://github.com/termux/termux-packages/blob/33b1397f3699c4ac6f0d3567765571cee00679b0/scripts/build/termux_step_massage.sh#L74), which currently doesn't apply to subpackages, so I put in this softer check that at least one subpackage file must be installed.

I think this is a good check generally for subpackages, let me know what you think.